pkg/metautil/metafile.go

```diff
@@ -638,9 +657,6 @@ func (writer *MetaWriter) ArchiveSize() uint64 {
 	for _, file := range writer.backupMeta.Files {
 		total += file.Size_
 	}
-	for _, size := range writer.metafileSizes {
```

Comment:
I'm wondering why remove this...? Seems the `for file in writer.backupMeta.Files` for backupmeta V1, and this for loop only for V2. It seems when using V1, `writer.metafileSizes` wouldn't even be accessed.

Reply:
@YuJuncen  Field `metafileSizes` seems the size of the inner index files, which should not be counted, I think only the size of the data files should be counted. If we count the index file size in backup, how we count their size when we only restore parts of the backup data.

Reply:
For the data files, `metafaleSizes` was accumulated here.
https://github.com/pingcap/br/blob/52c2bbae806207cd0c96091b89eef66d2e6703e0/pkg/metautil/metafile.go#L614-L615

Where the `writer.metafiles.size` would be collected by this:
https://github.com/pingcap/br/blob/52c2bbae806207cd0c96091b89eef66d2e6703e0/pkg/metautil/metafile.go#L394-L399

Let `op` be `AppendDataFile`, the `size` should be the file size:
https://github.com/pingcap/br/blob/52c2bbae806207cd0c96091b89eef66d2e6703e0/pkg/metautil/metafile.go#L351-L357

So, maybe only the `metafileSizes["metafile"]` is redundant? Also cc @3pointer.

Reply:
I think there is another issue that in meta v1, we only count the data file size, and in meta v2 we count data file size, schema size and Ddls size. It may cause inconsistency behavior.

Reply:
Well, so I guess `metafileSizes["datafile"]` is enougth?